rest/route/github_agent_test.go

```diff
@@ -0,0 +1,77 @@
+package route
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/mock"
+	"github.com/evergreen-ci/gimlet"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateInstallationToken(t *testing.T) {
+	validOwner := "owner"
+	validRepo := "repo"
+
+	for tName, tCase := range map[string]func(ctx context.Context, t *testing.T, gh *createInstallationToken, env *mock.Environment){
+		"parse should error on empty owner and repo": func(ctx context.Context, t *testing.T, handler *createInstallationToken, env *mock.Environment) {
```

Comment:
nit: I haven't commented on this a lot in other PRs, but can we stick to one convention for test names (seems like most of them use `PascalCase`, as does the top-level function. The problem with using spaces is that `go test` auto-translates spaces to underscores, so it both looks inconsistent with the other tests (it looks like `TestCreateInstallationToken/parse_should_error_on_empty_owner_and_repo` whereas most of the tests `TestFunctionName/TestsLookLikeThis`) and you have to know that if you're grepping for this particular test name, you have to translate all the underscores back to spaces.

I can add a mention to the style guide about this.